modules/network-observability-flow-filter-parameters.adoc

```diff
@@ -0,0 +1,59 @@
+:_mod-docs-content-type: REFERENCE
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+[id="network-observability-flowcollector-flowfilter-parameters_{context}"]
+= Flow filter configuration parameters
+The flow filter rules consist of mandatory and optional parameters.
+
+.Required configuration parameters
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`enable`
+| Set `enable` to `true` to enable eBPF flow filtering feature.
+
+|`cidr`
+| Provides the IP address and CIDR mask for the flow filter rule, supports IPv4 and IPv6 address format. If you want to match against any IP, you can use `0.0.0.0/0` or `::/0` for IPv4 and IPv6 respectively.
+
+|`action`
+| Describes the action to be taken for the flow filter rule. The possible values are `Accept` or `Reject`.
```

Comment:
```suggestion
| Describes the action taken by the flow filter rule. The possible values are `Accept` or `Reject`.
```
Trying to keep things in present tense, per the [Red Hat style guide](https://redhat-documentation.github.io/supplementary-style-guide/).